frontends/concrete-python/concrete/fhe/compilation/module.py

```diff
@@ -187,6 +199,10 @@ def run(
             Union[Value, Tuple[Value, ...]]:
                 result(s) of evaluation
         """
+
+        if self.configuration.simulate_encrypt_run_decrypt:
```

Comment:
no need to explain? like "# In simulate_encrypt_run_decrypt:, we skip encryption and decryption, and run will be normal python execution